.eslintrc.json

```diff
@@ -26,89 +33,24 @@
     "babelOptions": {
       "babelrc": false,
       "configFile": false,
-      "presets": [
-        "@babel/preset-env"
-      ],
-      "plugins": [
-        "@babel/plugin-syntax-jsx"
-      ]
+      "presets": ["@babel/preset-env"],
+      "plugins": ["@babel/plugin-syntax-jsx", "@babel/plugin-proposal-export-default-from"]
     },
     "ecmaVersion": 2020,
     "ecmaFeatures": {
       "jsx": true
     }
   },
-  "plugins": [
-    "react",
-    "react-hooks",
-    "sonarjs",
-    "import",
-    "prettier"
-  ],
+  "plugins": ["react", "sonarjs", "import"],
```

Comment:
Why removing react-hooks linter rules? @alorle

Reply:
They are defined by the `extends` property. The [docs](https://www.npmjs.com/package/eslint-plugin-react-hooks) talk about `plugins` section only for avance usage.

Reply:
Seeing your comment, I have a doubt about the react plugin (not the react-hooks one) and the same thing happens, so I have also removed it from the list of plugins.